### DISCUSSION TITLE
Have mini.pick help open help files in vertical split by default

### DISCUSSION THREAD

**Discussion by diego-velez** [nvim-mini/mini.nvim#1421] [Q&A] [answered]
Basically the title.

I tried this, but got some weird behavior.
```
        require('mini.pick').builtin.help({}, {
          mappings = {
            choose = '',
            choose_in_vsplit = '<CR>',
          },
        })
```

**Comment by Rishabh672003** [comment]
```lua
local vert_help = function(selected_item)
  vim.api.nvim_win_call(MiniPick.get_picker_state().windows.target, function()
    vim.cmd("vert help ".. selected_item.name)
	MiniPick.set_picker_target_window(vim.api.nvim_get_current_win())
    return true
  end)
end
MiniPick.builtin.help({}, { source = { choose = vert_help } })
```

**Reply by echasnovski** [reply]
This is one way to do it. Although needs `MiniPick.set_picker_target_window(vim.api.nvim_get_current_win())` call before `return true` for help window to be in focus after picking.

It also seems like a lot of code for what looks like a reasonable workflow. I am thinking about adding special option for default split direction.

**Comment by echasnovski** [comment, marked as answer]
On the latest `main` there is now [`default_split` local option](https://github.com/echasnovski/mini.nvim/blob/7b210cc5207e36f562f6c1e83200d5dfc0a4451d/doc/mini-pick.txt#L1144-L1145). So now making `<CR>` open in vertical split is as easy as `:Pick help default_split='vertical'`. Thanks for the great question!

**Reply by diego-velez** [reply]
That was fast! You're a legend 😎
